include/belle.hh

```diff
@@ -1817,6 +1818,25 @@ public:
     return _attr->public_dir;
   }
 
+  // overwrite the default index filename
+  Server& index_file(const std::string&& index_file_)
+  {
+     if (index_file_.empty())
+     {
+        _attr->index_file = "index.html";
```

Comment:
Since the index file is default initialized in the construction of the Attr object, setting it again if the parameter is empty is redundant.

```cpp
{
  if (! index_file_.empty())
  {
    _attr->index_file = index_file_;
  }

  return *this;
}
```

Reply:
If the user of the lib does something like this:
`OB::Belle::Server app{ip, port};
app.index_file("foo.html");
app.index_file("");`
(maybe with some boilerplate/indirections and variables between) then, in your version, "foo.hml" will be considered as index file. In my version it will be reseted to "index.hml".
It's something you need to decide.